aten/src/ATen/cuda/tunable/GemmHipblaslt.h

```diff
@@ -183,19 +183,19 @@ static size_t GetHipblasltWorkspaceSize() {
   // 256MB is max workspace size allowed for hipblaslt
   // hipblaslt-bench uses 32MB
   // recommendation from hipblaslt author was 76MB
-  size_t workspace_size = 2*128*1024*1024; // default 256MB
+  size_t workspace_size = 32*1024;  // going with 32MB
```

Comment:
maybe calling this constant: DEFAULT_HIPBLASLT_WORKSPACE_SIZE_IN_MB ?

Reply:
won't fix.